bitseq/sequence.go

```diff
@@ -313,14 +319,14 @@ func (h *Handle) set(ordinal, start, end uint64, any bool, release bool, serial
 		curr := uint64(0)
 		h.Lock()
 		store = h.store
-		h.Unlock()
 		if store != nil {
+			h.Unlock() // The lock is acquired in the GetObject
```

Comment:
It appears `store.GetObject` will modify `h` (through the `SetValue` and `SetIndex` calls). If so, shouldn't we keep `h` locked while that is happening?